### DISCUSSION TITLE
Xstate Viz throw error for send actions creator?

### DISCUSSION THREAD

**Discussion by raitucarp** [statelyai/xstate#2142] [General]
this code throw me an error, when I implement send actions creator:

```javascript
const sendName = send((context, event) => ({
  type: 'NAME',
  name: context.user.name
}));

...
actions: [
  assign({
    data: (context, event) => ({
      ...context.data,
      name: event.target.value,
    }),
  }),
  sendName ,
],
....
```
Related issue:

https://github.com/statecharts/xstate-viz/issues/81

**Comment by davidkpiano** [comment]
The visualizer is using an outdated version of XState.

Please use the [XState inspector (CodeSandbox template)](https://codesandbox.io/s/xstate-viz-template-f3dtr) instead.

More info: https://xstate.js.org/docs/packages/xstate-inspect/

**Reply by raitucarp** [reply]
Ok, thanks.
